test-suite/unit-tests/dune

```diff
@@ -0,0 +1,43 @@
+(subdir lib
```

Comment:
What does this file have to do with this PR?

Reply:
oops slipped in when rebasing against test-suite PR.